src/view/container/creation_page.rs

```diff
@@ -713,7 +754,16 @@ impl CreationPage {
         let create_opts = if cmd.is_empty() {
             create_opts
         } else {
-            create_opts.command([cmd.as_str()])
+            let args = imp
+                .cmd_args
+                .borrow()
+                .to_owned()
+                .to_typed_list_model::<model::CmdArg>()
+                .into_iter()
+                .map(|arg| arg.arg_take());
```

Comment:
Even if you now notify, taking the argument could be critical.
Imagine a user wants to create a container. The user then chooses a name that is already taken by another container. Now, when the user corrects the mistake, two things can happen:

1. The user might not notice that his arguments are in the Nexus now.
2. The user notices that the arguments are gone, but he might become frustrated for filling them in again

demonstration

https://user-images.githubusercontent.com/3630213/192161130-9d1ef3d1-75ee-46bb-8e9e-4adb24d1da0e.mp4

```suggestion
                .map(|arg| arg.arg());
```

Reply:
Alright, now I get it. Thank you for explaining. I guess I'll have to read some more docs for gtk-rs to get a grasp of notify and the whole argument management.